docs/user-guide/README.md

```
@@ -69,3 +69,15 @@
 - [Hook 系统设计](../hook-system-design.md) — 前置/后置 hook 机制
 - [Session 管理设计](../session-management-design.md) — Session 抽象由来
 - [架构总览](../ARCHITECTURE_HEALTH_CHECK_REPORT.md) — 后端模块划分
+
+---
+
+##变更日志
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)